.travis.yml

```diff
@@ -11,44 +11,44 @@ before_install:
 
 jobs:
     include:
-        - name: "linux-x86-clang-locks"
+        - name: "linux-x86-clang"
           os: linux
           compiler: clang
-          env: CONFIG_OPTS="--enable-swtok --enable-icsftok --enable-ccatok --enable-testcases --enable-locks --with-systemd" CFLAGS="-O3 -Werror -DDEBUG"
+          env: CONFIG_OPTS="--enable-swtok --enable-icsftok --enable-ccatok --enable-testcases --with-systemd" CFLAGS="-O3 -Werror -DDEBUG"
         - name: "linux-x86-gcc-tm"
           os: linux
           compiler: gcc
-          env: CONFIG_OPTS="--enable-swtok --enable-icsftok --enable-ccatok --enable-testcases --disable-locks --with-systemd" CFLAGS="-O3 -Wno-clobbered -Werror"
-        - name: "linux-ppc64le-clang-locks"
+          env: CONFIG_OPTS="--enable-swtok --enable-icsftok --enable-ccatok --enable-testcases --with-systemd" CFLAGS="-O3 -Wno-clobbered -Werror"
```

Comment:
As we know that tm is no more available, we could remove the "xxx-tm" jobs from Travis. However, there is one difference between the previous "-locks" job compared to -tm:
-Werror versus -Wno-clobbered -Werror
Are you intentionally leave the -tm job because of this?

Reply:
well, I guess I should remove the '-tm' from the names of the jobs. But I want to keep these jobs in general, because they are compiling with gcc versus the others use clang. I would like to cover both compilers as they sometimes behave differently in regards to warnings.